plugins/pyenv/pyenv.plugin.zsh

```diff
@@ -3,32 +3,6 @@
 
 # Load pyenv only if command not already available
 if command -v pyenv &> /dev/null && [[ "$(uname -r)" != *icrosoft* ]]; then
-    FOUND_PYENV=1
-else
-    FOUND_PYENV=0
-fi
-
-if [[ $FOUND_PYENV -ne 1 ]]; then
-    pyenvdirs=("$HOME/.pyenv" "/usr/local/pyenv" "/opt/pyenv" "/usr/local/opt/pyenv")
-    for dir in $pyenvdirs; do
-        if [[ -d $dir/bin ]]; then
-            export PATH="$PATH:$dir/bin"
-            FOUND_PYENV=1
-            break
-        fi
-    done
-fi
-
-if [[ $FOUND_PYENV -ne 1 ]]; then
-    if (( $+commands[brew] )) && dir=$(brew --prefix pyenv 2>/dev/null); then
-        if [[ -d $dir/bin ]]; then
-            export PATH="$PATH:$dir/bin"
-            FOUND_PYENV=1
-        fi
-    fi
-fi
-
-if [[ $FOUND_PYENV -eq 1 ]]; then
     eval "$(pyenv init - --no-rehash zsh)"
```

Comment:
I think after recent update to `pyenv` also this line should be changed to

```sh
eval "$(pyenv init --path --no-rehash zsh)"
```

Reply:
See my recent push.